packages/codemirror/style/base.css

```diff
@@ -151,3 +151,7 @@
   opacity: 1;
   transition-delay: 0s;
 }
+
+.cm-builtin {
+  color: var(--jp-mirror-editor-builtin-color);
+}
```

Comment:
This rule will be better set in the Jupyter codemirror theme at https://github.com/jupyterlab/jupyterlab/blob/e40ed2ecd299e70c95dea1cd4ef233dfd42cee92/packages/codemirror/src/theme.ts#L15